File: abc/abc230/a.cs
// https://atcoder.jp/contests/abc230/submissions/46467286

using System;

class A
{
  static void Main()
  {
    int n = int.Parse(Console.ReadLine());

    Console.Write("AGC0");
    if(n < 10)
    {
      Console.Write(0);
    }

    if(n >= 42)
    {
      Console.WriteLine(n + 1);
    }
    else
    {
      Console.WriteLine(n);
    }
  }
}

File: abc/abc230/a.rs
// https://atcoder.jp/contests/abc230/submissions/47524681

use proconio::input;

fn main() {
    input! {
        n: u8
    }

    print!("AGC0");
    if n < 10 {print!("{}", 0);}
    print!("{}", if n < 42 {n} else {n + 1});
}

